### PR TITLE
Grammatical change

### PR DESCRIPTION
**Pull request by akshatcoder-hash**
The words *clear, actionable* were replaced by *clear and actionable* in the ReadMe.

### CODE REVIEW

**Comment by windsonsea** [comment]
Looks more official for two juxtaposed words though many people like to write in this way to emphasize something.

/lgtm

**Comment by divya-mohan0209** [comment]
Hello @akshatcoder-hash , are you comfortable checking the entire page for any inconsistencies? Please let us know if you'd be able to work on this further since IMO this doesn't really add any value/clarity to the page.

**Comment by akshatcoder-hash** [comment]
@divya-mohan0209 . That would be great!

File: README.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ You can also reach the maintainers of this project at:
 
 You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account. This copy is called a _fork_. Make any changes you want in your fork, and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
 
-Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear, actionable feedback. As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
+Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear and actionable feedback. As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
```
Comment (reylejano) [inline README.md:157]: Personally, I prefer the original as it's more concise
```suggestion
Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear, actionable feedback. As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
```

**Comment by reylejano** [comment]
I'm closing this PR as there has been no traction and this PR is a trivial change and does not add value/clarity to the page as mentioned in this [comment](https://github.com/kubernetes/website/pull/34814#pullrequestreview-1028481759) 
/close

**Pull request closed without merging**